mxnet-bot/LabelBotFullFunctionality/LabelBot.py

```diff
@@ -265,7 +265,8 @@ def parse_webhook_data(self, event):
                     logging.error('Repo labels: {}'.format(str(set(self.all_labels))))
                     raise Exception("Provided labels don't match labels from the repo")
 
-                action = payload["comment"]["body"].split(" ")[1]
+                # Allows @mxnet-label-bot to be specified anywhere in the comment body and still be referenced
+                action = payload["comment"]["body"][payload["comment"]["body"].index("@mxnet-label-bot"):].split(" ")[1]
```

Comment:
At this point I really would like to have a unittest. This is getting too frizzy for me.

Reply:
Included more extensive unit tests